app/services/usps_in_person_proofing/proofer.rb

```diff
@@ -5,23 +5,34 @@ class Proofer
     AUTH_TOKEN_CACHE_KEY = :usps_ippaas_api_auth_token
     # Automatically refresh our auth token if it is within this many minutes of expiring
     AUTH_TOKEN_PREEMPTIVE_EXPIRY_MINUTES = 1.minute.freeze
+    # Sponsor ID to use for enhanced in-person proofing
+    USPS_EIPP_SPONSOR_ID = 36
```

Comment:
can this be something that lives in config values so we don't have it in source code?

Reply:
Agreed.  I'm also going to change it from 36 to '36', which is consistent with using a string value for `usps_ipp_sponsor_id`.